checkstyle/import-control.xml

```diff
@@ -93,13 +93,19 @@
 
     <subpackage name="record">
       <allow pkg="net.jpountz" />
+      <allow pkg="org.apache.kafka.common.header" />
```

Comment:
Since we now only have the `Header` and `Headers` classes public, maybe we could locate them under `common`?